python_modules/dagster/dagster/_core/storage/tags.py

```diff
@@ -57,6 +57,7 @@
 MAX_RUNTIME_SECONDS_TAG = f"{SYSTEM_TAG_PREFIX}max_runtime"
 
 AUTO_MATERIALIZE_TAG = f"{SYSTEM_TAG_PREFIX}auto_materialize"
+AUTOMATION_CONDITION_TAG = f"{SYSTEM_TAG_PREFIX}automation_condition"
```

Comment:
naming-wise, one might think the value here would be the name of the automation condition rather than a boolean indicating it originated from an automation condition